django/db/backends/postgresql/features.py

```diff
@@ -70,6 +70,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_update_conflicts = True
     supports_update_conflicts_with_target = True
     supports_covering_indexes = True
+    supports_computed_field = True
+    supports_computed_field_params = True
+    supports_computed_field_virtual = False
```

Comment:
Postgres only supports `GENERATED ALWAYS ... STORED` columns, not `GENERATED ALWAYS ... VIRTUAL`